src/GraphQL.Tests/Subscription/SubscriptionSchema.cs

```diff
@@ -23,14 +23,14 @@ public class ChatSubscriptions : ObjectGraphType
     public ChatSubscriptions(IChat chat)
     {
         _chat = chat;
-        AddField(new FieldType
+        AddField(new SubscriptionRootFieldType
         {
             Name = "messageAdded",
             Type = typeof(MessageType),
             StreamResolver = new SourceStreamResolver<Message>(Subscribe)
         });
```

Comment:
As we discourage this design style, this doesn't seem applicable to understanding typical required changes.

Reply:
This is the minimum change for the code to be compiled. I did not aim to rewrite everything.